mb-nft-v2/src/approvals.rs

```diff
@@ -37,17 +37,17 @@ impl MintbaseStore {
     #[payable]
     pub fn nft_approve(
         &mut self,
-        token_id: U64,
+        token_id: String,
```

Comment:
What is the reason for this change?

Reply:
Previously token IDs were just a stringified `u64`, now they are `${metadata_id}:${token_id}`, so keeping the `U64` here would fail.